lms/templates/credit_notifications/credit_eligibility_email.txt

```diff
@@ -9,8 +13,7 @@ ${dashboard_link}
 
 ${_(u"Course credit can help you get a jump start on your university degree, finish a degree already started, or fulfill requirements at a different academic institution.")}
 
-${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. No application, transcript, or grade report is required.')}
+${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. After you purchase credit you will have access to your official academic transcript from Arizona State University.')}
```

Comment:
@marcotuts Nope. Since the relation between course and its partners is saved in ecommerce.
I think text suggestion by @srpearce is good for our case.